personio-personnel-data-api-oa3.yaml

```diff
@@ -3252,6 +3252,11 @@ components:
         skip_approval:
           type: boolean
           description: "Optional, default value is true. If set to false, the approval status of the attendance period will be \"pending\" if an approval rule is set for the attendances type. The respective approval flow will be triggered."
+        project_id:
+          type: integer
+          format: int32
+          description: ID of a Project
```

Comment:
Can we name it like `The ID of the project ` ... `ID of a Project` could refer to `a` project other than this one.

Reply:
Changed it to the suggestion 👍